acceptance/User.Specs/Features/Results.feature

```diff
@@ -13,8 +13,9 @@ Scenario: SomeResults
   Given I have performed a search for records that exist
   Then I am on the results page
   And the original search parameter is shown
-  And the results contain the name
-  And the results contain the address
-  And the results contain the postcode
-  When I click the result
-  Then I am taken to the sharing report
+  And there are 2 results
+  And the results contain the name "NHS VEGETAL"
+  And the results contain the address "BEEF WELLINGTON PLACE"
+  And the results contain the postcode "LS2 5AP"
+  When I click result 2
+  Then I am taken to the detail page
```

Comment:
Do we need to extend this to test pagination, we can set the page size to be 1 result if needed. Or is this too low level?

Reply:
Do we need to check verify the back to search results functionality?

Reply:
> page size

AFAICT the code currently doesn't support changing the pagination size at all, let alone using the UI.
Do we need a UI feature for this? Is it possible it will remain unused in a database with ~ 10,000 entries and reasonably specific search criteria (when the default page size is 100)

> Do we need to check verify the back to search results functionality?

Yes, and also for the back-to-search-box as well.

Reply:
Implemented these two bits as steps in existing test cases